setup.cfg

```diff
@@ -24,13 +24,13 @@ classifiers =
 [options]
 packages = find:
 setup_requires = setuptools_scm
-python_requires = >=3.8
+python_requires = >=3.11
 install_requires =
-    astropy~=4.2
-    lstchain~=0.9.13
-    ctapipe~=0.12.0
-    matplotlib~=3.5
-    numpy<1.22.0a0
+    astropy~=5.0
+    lstchain==0.10.0
+    ctapipe~=0.19.2
```

Comment:
let's remove ctapipe and protobuf, which are not direct dependencies of lstosa, and they will be installed with lstchain